### PR TITLE
Add xprop tool for gtk-dark-theme extension

### PR DESCRIPTION
**Pull request by ai**
Fixes https://github.com/flathub/io.atom.Atom/issues/53

### CODE REVIEW

File: io.atom.Atom.json [hunk 1]
```diff
@@ -145,6 +145,17 @@
...
+        "name": "xprop",
+        "sources": [
+            {
+                "type": "archive",
+                "url": "https://xorg.freedesktop.org/archive/individual/app/xprop-1.2.3.tar.bz2",
+                "sha256": "d22afb28c86d85fff10a50156a7d0fa930c80ae865d70b26d805fd28a17a521b"
+            }
+        ]
     }
+},
```
Comment (TingPing) [inline io.atom.Atom.json]:
```suggestion
```

**Comment by TingPing** [comment]
Change commit to something useful:

`Add xprop tool for gtk-dark-theme extension`

**Comment by ai** [comment]
> Change commit to something useful:

Can I ask you with squashing so GitHub will use PR title (I renamed PR to a better name)

**Comment by TingPing** [comment]
bot, build

**Comment by TingPing** [comment]
`xprop` probably installs a few things that need to be added to `cleanup` such as man pages. Since you clearly didn't build this locally its probably a bit hard to guess them...

Maybe @cpba can follow up.

**Comment by cpba** [comment]
I'll try to remember to take a look when I get home.

**Comment by cpba** [comment]
As far as I can tell it only installs the man page and the binary, everything else is under debug.

**Comment by cpba** [comment]
bot, build

**Comment by cpba** [comment]
@ai could you please install and test this build:

    flatpak install --user https://dl.flathub.org/build-repo/4364/io.atom.Atom.flatpakref

To make sure that it does indeed fix your issue?

**Comment by ai** [comment]
Everything works



(Wow, this feature of building apps from PR is awesome)

**Comment by ai** [comment]
Should we wait for next Atom release before releasing this fix?

**Comment by cpba** [comment]
@ai It'll be up on flathub in 3 hours.

Unless somebody manually clicks on the publish button before that, then it'll be there earlier.

**Pull request merged**